aws/resource_aws_ses_configuration_set_test.go

```diff
@@ -97,9 +73,34 @@ func testAccCheckAwsSESConfigurationSetExists(n string) resource.TestCheckFunc {
 	}
 }
 
-var escRandomInteger = acctest.RandInt()
-var testAccAWSSESConfigurationSetConfig = fmt.Sprintf(`
+func testAccCheckSESConfigurationSetDestroy(s *terraform.State) error {
+	conn := testAccProvider.Meta().(*AWSClient).sesConn
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "aws_ses_configuration_set" {
+			continue
+		}
+
+		_, err := conn.DescribeConfigurationSet(&ses.DescribeConfigurationSetInput{
+			ConfigurationSetName: aws.String(rs.Primary.ID),
+		})
+
+		if err == nil {
+			if aerr, ok := err.(awserr.Error); ok && aerr.Code() == "ConfigurationSetDoesNotExist" {
+				return fmt.Errorf("The configuration set still exists")
```

Comment:
if the if-logic @ L89 is true, we can `return nil` in this case as it validates that the Resource has been deleted as expected

Reply:
in this resources test file, the Destroy method is a good example to refer to as well https://github.com/terraform-providers/terraform-provider-aws/blob/bafd3e6f90f63049ebf3acbc910f660d79b6f21c/aws/resource_aws_cognito_resource_server_test.go#L140